content/en/docs/reference/config/analysis/ist0150/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: ExternalNameServiceTypeInvalidPortName
+layout: analysis-message
+owner: istio/wg-user-experience-maintainers
+test: no
+---
+
+This message occurs for services of type ExternalName when the port doesn't follow the [Istio service port naming convention](/docs/ops/deployment/requirements/), the port is unnamed or the port is named tcp.
```

Comment:
Is this the best link? Maybe it should point the `Protocol Selection` page or maybe the `Explicit protocol collection` link.

Reply:
added it